client/explore-playlists/explore-playlists.html

```diff
@@ -1,3 +1,14 @@
-<div>
-  <h3>ALL playlist</h3>
+<div class="container" ng-controller="playlist-ctrl">
```

Comment:
not necessary to ad directive ng-controller here,
plus do not change the format of controllers